js/tji-datasets-explore.js

```diff
@@ -29,7 +29,8 @@ var COLOR_TJI_DEEPPURPLE = '#2D1752';
 
 var TJIGroupByBarChart = function(props) {
   this.$container = props.$container;
-  this.groupBy = props.groupBy;
+  this.group_by = props.group_by;
+  this.sort_by = props.sort_by || {column: 'key', direction: 'asc'};
```

Comment:
Couldja update the comments above to include this props object and explain it?

Reply:
👍